### DISCUSSION TITLE
Interaction with map only after one click or tap

### DISCUSSION THREAD

**Discussion by plumkewe** [tomickigrzegorz/leaflet-examples#75] [General]
How can I achieve this? What I want to do is the map stays inactive so I can scroll on my website, but when I click (or tap) on it, it becomes active so I can interact with it, zoom, click on buttons, and so on.

**Comment by tomickigrzegorz** [comment]
The easiest way is to add a layer above the map, preferably a transparent layer.
Clicking on this layer deletes it and then you have access to zoom, click on buttons, etc.

Take a look at this too - [disable-leaflet-interaction-temporary](https://gis.stackexchange.com/questions/54454/disable-leaflet-interaction-temporary)

**Reply by plumkewe** [reply]
huge thanks!
